### DISCUSSION TITLE
Lazy sync or Update causes fiendly-snippets error

### DISCUSSION THREAD

**Discussion by cdenneen** [NvChad/NvChad#2883] [General]
![Screenshot 2024-06-08 at 10 13 41 AM](https://github.com/NvChad/NvChad/assets/720097/df2abc1d-53ba-48d7-9242-3ae00a9d1bd5)

**Comment by siduck** [comment]
@cdenneen   just press x and I

**Reply by cdenneen** [reply]
@siduck yes but it's everytime I update or sync?

**Reply by siduck** [reply]
@cdenneen thats strange,  you sure you dont edit it manually?

**Comment by cdenneen** [comment]
Clean clone of starter with no modifications

**Reply by siduck** [reply]
idk then, only thing i can think of is `rm -rf .local/share/nvim`

**Reply by cdenneen** [reply]
Yeah did that too. Curious if there are other cache paths other than that one that could be left around and used that I need to clear as well? Do you happen to know possibly other paths nvim checks for cache?

**Reply by siduck** [reply]
~/.cache/nvim
